Wrappers/Python/cil/framework/framework.py

```diff
@@ -2915,15 +3097,52 @@ def subset(self, dimensions=None, **kw):
             return temp
 
     def get_slice(self,channel=None, angle=None, vertical=None, horizontal=None, force=False):
-        '''
-        Returns a new dataset of a single slice of in the requested direction. \
-        '''
-        try:
-            geometry_new = self.geometry.get_slice(channel=channel, angle=angle, vertical=vertical, horizontal=horizontal)
-        except ValueError:
-            if force:
-                geometry_new = None
-            else:
+        """
+        Returns a new AcquisitionData containing containing data sliced in the requested directions.
+        If the requested geometry can not be computed an error is raised. This behaviour can be changed with the 'force' argument.
+        
+        Slices can be in any combination of:
+            <dimension label> = index
+            <dimension label> = slice(index0,indexn,step)
+            <dimension label> = [index0,...,indexn]
+
+        Supports fancy indexing.
+  
+        Parameters
+        ----------
+        channel : int, list, slice
+            The requested slices in the channel 'direction' 
+        angle : int, list, slice
+            The requested slices in the angle 'direction' 
+        vertical : int, list, slice, str
+            The requested slices in the vertical 'direction', additionally supports special value 'centre'\
+            which returns centre slice of the dataset, interpolated if there is an even number of rows.
+        horizontal : int, list, slice
+            The requested slices in the horizontal 'direction', only symmetric slice objects will return a geometry 
+        force : boolian, default = False 
+            If the sliced geometry cannot be computed returns a DataContainer of sliced data
+
+        Notes
```

Comment:
Notes and Note are different for docs. 
If you want a text with margin to attract the reader, we shoudl use Note

Reply:
changed